pom.xml

```diff
@@ -269,6 +264,17 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
```

Comment:
This could be redundant, right?
https://github.com/fabric8io/cico-bash-library/blob/master/utils.sh#L89